index.html

```diff
@@ -1,38 +1,41 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="style.css" />
-    <title>Awesome-Book</title>
-  </head>
-  <body>
-    <h1>Awesome-Book</h1>
-    <header>
-    </header>
-    <div class="container">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+  <title>Awesome-Book</title>
+</head>
```

Comment:
- [ ] Your application is doing great! Just a friendly suggestion: it would be fantastic if you could spend a little more time on the styling aspect, ensuring it matches the wireframe as one of [the requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m2_plain_js_classes_v1_1.md). Keep up the good work!

   ### Given wireframe:
   ![image](https://github.com/MohammadYaser/Awesome-Books/assets/107952724/3af4c819-6c45-4372-a7f5-830e42826be9)

   ### Your design:
   ![image](https://github.com/MohammadYaser/Awesome-Books/assets/107952724/478f7aa7-2948-4d16-ac64-1e590a8dffb3)